common/src/main/java/io/druid/concurrent/Execs.java

```diff
@@ -55,6 +55,11 @@ public static ThreadFactory makeThreadFactory(String nameFormat)
     return new ThreadFactoryBuilder().setDaemon(true).setNameFormat(nameFormat).build();
   }
 
+  public static ExecutorService threadPool(String nameFormat)
```

Comment:
Its actually not needed in this PR anymore, I'll remove it. But it probably should have been cachedThreadPool